sdcm/logcollector.py

```diff
@@ -1297,6 +1297,17 @@ def collect_logs(self, local_search_path: Optional[str] = None) -> list[str]:  #
         return [s3_link]
 
 
+class CertsCollector(BaseSCTLogCollector):
+    log_entities = [
+        DirLog(name='ssl_conf/*', search_locally=True),
+    ]
+    cluster_log_type = 'ssl-conf'
+    cluster_dir_prefix = 'ssl-conf'
+
+    def collect_logs(self, local_search_path: Optional[str] = None) -> list[str]:
+        return super().collect_logs(local_search_path=local_search_path)
```

Comment:
Why is it needed?

Reply:
Removed. 
I was looking at/repeated another k8s related collector, but It is not needed indeed.